infra/create-cluster/create-cluster.sh

```diff
@@ -29,8 +31,17 @@ if ! [[ "${CNAME}" =~ ${ALLOWED_NAMES} ]]; then
     exit 1
 fi
 
+function infractl_call() {
+    local endpoint_options=("--endpoint" "$ENDPOINT")
+    if [ "$INSECURE" = "true" ]; then
```

Comment:
I wish ACS had a bash coding standard that said no single `=` in comparison and use `[[` over `[` but we don't and there is no technical reason for one over the other so I can't complain about this.

Reply:
As long as shellcheck doesn't complain, I see no reason to change :D 
In honesty, I tried to keep the same style within this file.